README.md

```diff
@@ -30,7 +30,193 @@ Attributes
     <td>log directory</td>
     <td><tt>/var/log/pgpool</tt></td>
   </tr>
+<tr>
```

Comment:
I can drop an issue against the readme to get it reworked from its original format, for the purposes of this PR though imo this particular item can be pushed to a future iteration.